guide/contribute/contribute.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: guide
-title: Contribute
+title: Contribute to Guide
```

Comment:
```suggestion
title: Contribute to guide
```

Reply:
This makes sense.
Does this mean we need to change: parent: .... as well for navigation to work?